_data/chains/eip155-183.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Ethernity Mainnet",
+  "chain": "Ethernity",
+  "rpc": ["https://mainnet.ethernitychain.io"],
```

Comment:
need to include faucets 

"faucets": [], - with or without link to request faucets is fine

Reply:
Thanks @macvts, I added the missing property